extensions/amp-ad-network-doubleclick-impl/0.1/flexible-ad-slot-utils.js

```diff
@@ -65,8 +65,13 @@ function getElementWidthVisitor(setWidth) {
         setWidth(0);
         return true;
       default:
-        // If no layout is provided, we must use getComputedStyle.
-        setWidth(parseInt(style.width, 10) || 0);
+        // If no layout is provided, we must use getComputedStyle. Padding is
+        // not included in the overall computed width, so we must manually
+        // include it.
+        const paddingLeft = parseInt(style.paddingLeft, 10) || 0;
+        const paddingRight = parseInt(style.paddingRight, 10) || 0;
+        const totalPadding = paddingLeft + paddingRight;
+        setWidth((parseInt(style.width, 10) || 0) + totalPadding);
```

Comment:
Just to confirm: do we need to worry about things like margins and borders here?

Reply:
Good catch. We do need to include borders. But not margins, as that's space "outside" of the element, loosely speaking.